### ISSUE TITLE
Failing to load a module does not fail the tests

### ISSUE THREAD

**Issue by jehon** [jasmine/jasmine#1865]
## Expected Behavior
Using jasmine under karma.

When loading modules, if one module has an error in it (example: it could not import another module due to a typo in the import statement), the test should fail:

Example code:
``` failingtest.js
import { something } from './somewhere.js';

describe...
```
If "somewhere.js" does not exists, the test should fail.

## Current Behavior
Today, the test pass, skipping the "failingtest.js" as it did not exists.
=> It shows:
```
09 11 2020 11:52:49.258:INFO [karma-server]: Karma v5.2.3 server started at http://localhost:9876/
09 11 2020 11:52:49.259:INFO [launcher]: Launching browsers ChromiumHeadless with concurrency unlimited
09 11 2020 11:52:49.263:INFO [launcher]: Starting browser ChromiumHeadless
09 11 2020 11:52:49.542:INFO [Chrome Headless 86.0.4240.183 (Linux x86_64)]: Connected on socket pCX5O55W4Zk2P-JAAAAA with id 38747259
==> 09 11 2020 11:52:49.592:WARN [web-server]: 404: /base/tests/something.js
Chrome Headless 86.0.4240.183 (Linux x86_64): Executed 1 of 1 SUCCESS (0.003 secs / 0.001 secs)
TOTAL: 1 SUCCESS
```

While obviously, the second test did not run and should be in error

## Possible Solution
When importing a test file, it should catch the error:
``` (pseudo code)
import('failingtest.js')
  .then(
      () => jasmine current behavior,
      () => the file failed to load, it should fail  the test
  ),
```

## Suite that reproduces the behavior (for bugs)
See https://github.com/jehon/example-jasmine for a bare minimal example

## Your Environment

* Version used: 3.6.3 (core 3.6.0)
* Environment name and version (e.g. ChromiumHeadless 86.0.4240.183 , node.js 4.14.0):
* Operating System and version : seen in Ubuntu 20
* Link to your project: https://github.com/jehon/example-jasmine

**Comment by sgravrock** [comment]
I'm mostly unable to get your example working on your machine. I saw it do what you describe twice (once with 3.6.3, and once after I rolled back to 3.6.1) but every other time I've gotten this:

```
% npm run test

> @ test /private/tmp/example-jasmine
> karma start karma.conf.js --single-run

09 11 2020 09:49:12.441:INFO [karma-server]: Karma v5.2.3 server started at http://localhost:9876/
09 11 2020 09:49:12.444:INFO [launcher]: Launching browsers ChromiumHeadless with concurrency unlimited
09 11 2020 09:49:12.450:INFO [launcher]: Starting browser ChromiumHeadless
09 11 2020 09:50:12.454:WARN [launcher]: ChromiumHeadless have not captured in 60000 ms, killing.
09 11 2020 09:50:14.457:WARN [launcher]: ChromiumHeadless was not killed in 2000 ms, sending SIGKILL.
09 11 2020 09:50:16.458:WARN [launcher]: ChromiumHeadless was not killed by SIGKILL in 2000 ms, continuing.
```

That makes it hard for me to investigate further.

In any case, it's not Jasmine that loads the specs in your case, it's Karma. I'm not anywhere near familiar enough with how Karma integrates with Jasmine to debug it. But my guess is that Karma isn't routing that particular kind of error to Jasmine's global error handler. (It looks like ES module load failures in the browser trigger the script tag's `error` event rather than the global `error` event.) I suggest reporting this to the Karma folks.

BTW, it looks like your example doesn't use the `jasmine` package, only `jasmine-core`. You can probably remove `jasmine` from the system that you based this off of, and keep only `jasmine-core`.

**Comment by jehon** [comment]
Thanks, I did reduce the package.json
I will report that to the karma team.

PS: do you have chrom**ium** installed? The karma config specify chromium. Otherwise, try to change line 12 from "ChromiumHeadless" to "ChromeHeadless"...

**Comment by jehon** [comment]
I confirm that using "jasmine", this is solved in jasmine 3.6.3 (with jasmine-core 3.6.0).

To see it working successfully, the same project https://github.com/jehon/example-jasmine with "npm run test-jasmine"

the bug in karma is still present and open.

Should we close this?

**Comment by sgravrock** [comment]
Yes, thanks.
